.editorconfig

```diff
@@ -5,68 +5,42 @@ root = true
 indent_style = space
 indent_size = 4
 max_line_length = 120
-
 end_of_line = lf
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
-[*.{ts, js}]
-indent_style = space
-indent_size = 4
-
-[*.{html}]
+[*.md]
 indent_style = space
-indent_size = 2
+max_line_length=off
+trim_trailing_whitespace = false
 
-[*.json]
+[*.{yaml,yml}]
 indent_style = space
+max_line_length=off
 indent_size = 2
-
-[*.md]
 trim_trailing_whitespace = false
 
-[*.php]
+[*.{ts, js}]
 indent_style = space
 indent_size = 4
 
+[*.json]
+indent_style = space
+indent_size = 2
```

Comment:
This is partly inherited from `*`, we can remove it.
Move it below all rules with `indent_size = 4`.